### ISSUE TITLE
Alarm sounds broken all of a sudden

### ISSUE THREAD

**Issue by edschindler** [LMS-Community/slimserver#823]
All of a sudden, my SB Radios won't play their established alarm sounds:
- Instead of my selected alarm sound, I get some default music clip (which I'm guessing is the built-in one)
- In alarm setup, attempting to play any of the alarm sounds results in silence (it appears to be playing)
- Music plays fine otherwise
 
This happens on more than one Radio, and happens after restarting the Radios and the server (service, I did not reboot the entire ubuntu server). I didn't see anything that caught my eye in the log file, either squeezebox server's or systemd's.

Logitech Media Server Version: 8.2.0 - 1627922070 @ Tue Aug 3 11:37:35 CEST 2021
Ubuntu 18.04 on amd64 with auto security updates

There was a Perl update a week or so ago, but it didn't correspond to when the alarm started failing, which just happened last night. (perl-base:amd64 (5.26.1-6ubuntu0.5, 5.26.1-6ubuntu0.6), libperl5.26:amd64 (5.26.1-6ubuntu0.5, 5.26.1-6ubuntu0.6), perl:amd64 (5.26.1-6ubuntu0.5, 5.26.1-6ubuntu0.6))

Thoughts?

**Comment by michaelherger** [comment]
This can happen when you'd be using an online source such as a radio station, but were suffering from a (temporary) network issue.

**Comment by edschindler** [comment]
The network seems OK, as I can play music stored on my local network. (That was the first thing I checked, because I know the Radios fall back to their built-in alarm sound if the server is out of communication.)

I just checked my squeezebox classic. That also produces its default sound (a beep, not music) rather than the selected alarm sound.

**Comment by michaelherger** [comment]
Hmm... The Classic wouldn't have any fallback sound built-in. It can only stream from a file LMS knows about. Would the web UI show a track as playing when this happens?

**Comment by edschindler** [comment]
The Classic shows "Backup Alarm" in the web playlist.

I just did a "test play" (in alarm setup) on the Radio, and the web playlist shows the name of the selected alarm sound, though no sound was being produced. I played a "real" song to reset the playlist (which it did, on device and web interface), then let an alarm go off. The Radio playlist and the web playlist both showed the name of my selected alarm sound, but the Radio emitted the default tune (after a pause).

(I've been using Foghorn as my selected alarm sound, but I couldn't get any of them to play in alarm setup.)

**Comment by edschindler** [comment]
I didn't realize until I browsed the perl code that the alarm sounds were stored at squeezebox.com, I thought they were local. I tried querying my selected alarm sound with my browser just now, and got the file. I tried wgetting it from the server and got the file. Finally I hit play on the alarm playlist and the alarm sound sounded.

http://content.mysqueezebox.com/static/sounds/effects/foghorn.mp3

So it appears that the squeezebox.com was down for a while and is now back up. So @michaelherger , you were completely right about network issue and on-line source, so thank you. I just didn't realize I was using an on-line source.
